packages/desktop-client/src/components/mobile/MobileAmountInput.js

```diff
@@ -0,0 +1,399 @@
+    if (this.props.onBlur) {
+      const absValue = Math.abs(value);
+      this.props.onBlur(this.state.isNegative ? -absValue : absValue);
+    }
+  };
+
+  render() {
+    const { textStyle, style, focusedStyle, buttonProps } = this.props;
+    const { focused } = this.state;
+
+    return (
+      <View>
+        <AmountInputWithContext
+          {...this.props}
+          ref={el => (this.amount = el)}
+          onBlur={this.onBlur}
+          focused={focused}
+          style={[
+            {
+              width: 80,
+              transform: [{ translateX: 6 }],
+              justifyContent: 'center',
+            },
+            style,
+            focusedStyle,
+            !focused && {
+              opacity: 0,
+              position: 'absolute',
+              top: 0,
+            },
+          ]}
+          textStyle={[{ fontSize: 15, textAlign: 'right' }, textStyle]}
+        />
+
+        <View>
+          {!focused && (
+            <Button
+              style={{
+                position: 'absolute',
+                right: 'calc(100% + 5px)',
+                top: '8px',
+              }}
+              onClick={this.toggleIsNegative}
+            >
+              {this.state.isNegative ? '-' : '+'}
+            </Button>
+          )}
+          <Button
+            onClick={this.onFocus}
+            // Defines how far touch can start away from the button
+            // hitSlop={{ top: 5, bottom: 5, left: 5, right: 5 }}
+            {...buttonProps}
+            style={[
+              buttonProps && buttonProps.style,
+              focused && { display: 'none' },
+              {
+                ':hover': {
+                  backgroundColor: 'transparent',
+                },
+              },
+            ]}
+            bare={true}
+          >
+            <View
+              style={[
+                {
+                  borderBottomWidth: 1,
+                  borderColor: '#e0e0e0',
+                  justifyContent: 'center',
+                  transform: [{ translateY: 0.5 }],
+                },
+                style,
+              ]}
+            >
+              <Text style={[{ fontSize: 15, userSelect: 'none' }, textStyle]}>
+                {amountToCurrency(Math.abs(this.props.value))}
+              </Text>
+            </View>
+          </Button>
+        </View>
+      </View>
+    );
+  }
+}
+
+export const FocusableAmountInput = forwardRef((props, ref) => {
+  return (
+    <AmountAccessoryContext.Consumer>
+      {context => (
+        // eslint-disable-next-line
+        <FocusableAmountInput_ ref={ref} {...props} context={context} />
+      )}
+    </AmountAccessoryContext.Consumer>
+  );
+});
+
+const styles = new CSSStyleSheet();
+/* eslint-disable rulesdir/typography */
+styles.replace(`
+inputContainer {
+  position: 'absolute',
+  top: 0,
+  left: 0,
+  bottom: 0,
+  right: 0,
+  opacity: 0,
+}
+
+inputContent {
+  flexDirection: 'row',
+  alignItems: 'stretch',
+}
+`);
```

Comment:
Can this be moved to inline styles like the rest of the app?

Reply:
Turns out we didn't need these at all (at least in my testing), so I've gotten rid of them entirely via https://github.com/actualbudget/actual/pull/1340/commits/01d491cdc8304a570a7c9f8ce9148ccfaf68d9cb 😄